dvc/stage/__init__.py

```diff
@@ -131,6 +139,7 @@ def __init__(
         self._stage_text = stage_text
         self._dvcfile = dvcfile
         self.desc = desc
+        self.meta = Meta()
```

Comment:
This is not the same as `meta` field, right? If so, could we change the name? It is really confusing otherwise.

Reply:
No, we don't keep `meta` around. It's only on `_stage_text`. Will change it to `RawData`/`raw_data`.